zipkin-server/src/test/java/zipkin2/server/internal/health/ITZipkinHealth.java

```diff
@@ -73,7 +73,7 @@
 
     // ensure we don't track health in prometheus
     assertThat(scrape())
-      .doesNotContain("health");
+      .doesNotContain("health_check");
```

Comment:
Is there any better string that only exposed by prometheus?
We have changed the meter tags to contain method and service name.
For example `scrape()` contains `method=getHealth` and `service=server.internal.health.ITzipkinHealth` which made this test failed. 😓

Reply:
but isn't this possibly a correct positive? why would we be adding metrics for the scrape endpoint? Maybe you can dump the actual contents here.